functions/openhab.js

```diff
@@ -424,6 +424,9 @@ function getThermostatItems(thermoGroup) {
 			if (tag === 'homekit:HeatingCoolingMode') {
 				values.heatingCoolingMode = member;
 			}
+			if (tag === 'CurrentHumidity') {
+				values.currentHumidity = member;
+			}
```

Comment:
Yes, mainly cause I havent got the chance to much test it with thermostat.
Makes sense!